paper-radio-group.html

```diff
@@ -157,34 +156,32 @@
       this.fire('paper-radio-group-changed');
     },
 
-    /**
-     * Selects the previous item. If the previous item is disabled, then it is
-     * skipped, and its previous item is selected
-     */
-    selectPrevious: function() {
-      var length = this.items.length;
-      var newIndex = Number(this._valueToIndex(this.selected));
-
-      do {
-        newIndex = (newIndex - 1 + length) % length;
-      } while (this.items[newIndex].disabled)
-
-      this._itemActivate(this._indexToValue(newIndex), this.items[newIndex]);
+    _activateFocusedItem: function() {
+      this._itemActivate(this._valueForItem(this.focusedItem), this.focusedItem);
     },
 
-    /**
-     * Selects the next item. If the next item is disabled, then it is
-     * skipped, and the next item after it is selected.
-     */
-    selectNext: function() {
-      var length = this.items.length;
-      var newIndex = Number(this._valueToIndex(this.selected));
+    _onUpKey: function(event) {
```

Comment:
IronMenubarBehavior's handles switching between `_focusNext` and `_focusPrevious` for RTL